docs/src/utils.js

```diff
@@ -0,0 +1,8 @@
+const uuidv4 = () => ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(
+    /[018]/g,
+    c => (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
+);
+
+const isProductionBuild = () => typeof window.__REACT_ERROR_OVERLAY_GLOBAL_HOOK__ === 'undefined';
```

Comment:
not sure it's the legit way of checking this, but it seems to work \o/

Reply:
It seems like this statement actually checks for React WebTools to be not available. Seems a bit hacky, but we can't be use `process.env.NODE_ENV` here, right?

Reply:
I don't think `process.env.NODE_ENV` works in the browser :/